src/components/Banner.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react';
+
+import { Box, Button, Typography } from '@mui/material';
+
+import { BANNER_CY, SUMMON_BUTTON_CY } from '../config/selectors';
+
+type Prop = {
+  title: string;
+  onSummonClick: () => void;
+  buttonDisable: boolean;
+};
+
+const bannerBox = {
+  height: '70px',
+  backgroundColor: '#BABABA',
+  padding: '2px',
```

Comment:
You could let mui use the theme values for example with: 
```ts
padding: 1,
```
you will get a padding of 4px (1 time the default spacing in MUI). so this way you can be consistant in your padding across the app. As you want !